tests/designs/vhdl_configurations/Makefile

```diff
@@ -34,8 +34,10 @@ else
 endif
 VHDL_SOURCES += $(COCOTB)/tests/designs/vhdl_configurations/configurations.vhd
 
-ifeq ($(SIM),$(filter $(SIM),ius xcelium))
-    SIM_ARGS += -v93
+ifneq (,$(SIM))
```

Comment:
I don't get this - is `SIM` ever blank?

Reply:
According to @ktbarrett, SIM is now blank if not specified by the user. It used to be defaulted to "icarus". Now, blank is equivalent to "icarus".